### PR TITLE
chore: prep release v0.11.0

### PR DESCRIPTION
**Pull request by Panaetius**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -18,6 +18,102 @@
...
+-  **core:** save and load workflow as jsonld
+   (`#1185 <https://github.com/SwissDataScienceCenter/renku-python/issues/1185>`__)
+   (`d403289 <https://github.com/SwissDataScienceCenter/renku-python/commit/d403289cec6cdb11ad002b17fbbb751de2818ce1>`__)
+-  **core:** separate models for migrations
+   (`#1431 <https://github.com/SwissDataScienceCenter/renku-python/issues/1431>`__)
+   (`127d606 <https://github.com/SwissDataScienceCenter/renku-python/commit/127d606fa9ad26479645a06b132a7c9a952ae989>`__)
+-  **dataset:** source and url for DatasetFile
+   (`#1451 <https://github.com/SwissDataScienceCenter/renku-python/issues/1451>`__)
+   (`b4fa5db <https://github.com/SwissDataScienceCenter/renku-python/commit/b4fa5dbed50b47a6a64f3c84b40bb99d86abd51e>`__)
+-  **init:** prompt for missing variables
```
Comment (jsam) [inline CHANGES.rst:86]: What is `init` scope?

File: CHANGES.rst [hunk 2]
```diff
@@ -18,6 +18,102 @@
...
+-  **service:** fixes handlers for internal loggers
+   (`#1433 <https://github.com/SwissDataScienceCenter/renku-python/issues/1433>`__)
+   (`a312f7c <https://github.com/SwissDataScienceCenter/renku-python/commit/a312f7c641c27de111fb78012c2f546405eee72e>`__)
+-  **service:** move project_id to query string on migrations check
+   (`#1367 <https://github.com/SwissDataScienceCenter/renku-python/issues/1367>`__)
+   (`0f89726 <https://github.com/SwissDataScienceCenter/renku-python/commit/0f8972698399aee50bced4a141cb24add6d9494b>`__)
+-  **tests:** integration tests
+   (`#1351 <https://github.com/SwissDataScienceCenter/renku-python/issues/1351>`__)
+   (`3974a39 <https://github.com/SwissDataScienceCenter/renku-python/commit/3974a39d5098ecebcc49f2d624e6912e1d423e03>`__)
+-  disable version check in githook calls
```
Comment (jsam) [inline CHANGES.rst:49]: is this core?

File: CHANGES.rst (hunk 1 above)
Comment (Panaetius) [reply CHANGES.rst]: changed to `cli`

File: CHANGES.rst (hunk 2 above)
Comment (Panaetius) [reply CHANGES.rst]: changed to `cli`

**Pull request merged**